### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by VikramjeetD**
- Setup Express, MongoDB, and react.
- `npm run dev` from root to run.
- Can get response from auth route (use Postman).

Replace `client/src` files with frontend.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
-  "name": "ChronoFactorem",
+  "name": "chronofactorem",
```
Comment (Aviral14) [inline package.json]: Why change the semantics of name?

File: package.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
   "version": "1.0.0",
   "description": "Timetable maker-helper",
-  "homepage": "",
-  "author": {
-    "name": "Dryft-bits",
-    "email": "",
-    "url": ""
-  },
-  "main": "app.js",
+  "homepage": "https://github.com/VikramjeetD/ChronoFactorem#readme",
```
Comment (Aviral14) [inline package.json]: make it https://github.com/Dryft-bits/ChronoFactorem#readme",

File: package.json (hunk 1 above)
Comment (VikramjeetD) [reply package.json]: That was somehow changed by VSCode. Changed it back.

File: client/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "axios": "^0.19.2",
+    "moment": "^2.24.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-moment": "^0.9.7",
+    "react-redux": "^7.1.3",
```
Comment (Aviral14) [inline client/package.json:14]: Have we decided to use Redux? @Kushagra-0801 @AetherPrior What say? Should we?

Comment (AetherPrior) [reply client/package.json:14]: Well does anyone here know how to use it?

Comment (VikramjeetD) [reply client/package.json:14]: @AetherPrior You're going to need either Redux or React Context API for managing the app state. Theres stuff to learn both ways, imo.

Comment (Kushagra-0801) [reply client/package.json:14]: We don't need axios and moment. I am not sure about Redux but I dont think we need it either

Comment (Kushagra-0801) [reply client/package.json:14]: @VikramjeetD IMHO, our state isn't so extensive that we need another library to help manage it.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    "debug": "^4.1.0",
+    "dotenv": "6.2.0",
+    "express": "^4.17.1",
+    "express-validator": "^6.4.0",
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^5.8.11",
+    "node-sass": "^4.5.3",
+    "node-sass-middleware": "^0.11.0",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3"
```
Comment (Kushagra-0801) [inline package.json:22]: Why are these in both client and here?

File: client/package.json (hunk 3 above)
Comment (VikramjeetD) [reply client/package.json:14]: @Kushagra-0801 
Axios might slightly simplify the code slightly. We could work without it as well though. For our purposes, using Axios and Redux wouldn't hurt though.

We don't need moment. Will remove.

File: package.json (hunk 4 above)
Comment (VikramjeetD) [reply package.json]: `npx create-react-app client` put that there by default. Not sure whether removing it would cause problems.

Comment (VikramjeetD) [reply package.json:22]: Not needed. Will remove with next commit.

**Pull request merged**